datacatalog/samples/v1beta1/datacatalog_search.py

```diff
@@ -52,9 +52,10 @@ def sample_search_catalog(include_project_id, include_gcp_public_datasets, query
         "include_project_ids": include_project_ids,
         "include_gcp_public_datasets": include_gcp_public_datasets,
     }
+    order_by = ""
```

Comment:
Not a valid value for `order_by` according to the protos above.

Reply:
I'll bring this up with the Data Catalog team. I don't know if they are still working with the person who originally wrote the sample config.